code/datums/action.dm

```diff
@@ -600,3 +600,33 @@
 		var/datum/language_holder/H = M.get_language_holder()
 		H.open_language_menu(usr)
 
+
+//Small sprites
+/datum/action/small_sprite
+	name = "Toggle Giant Sprite - Others will always see you as giant"
```

Comment:
Since i originally wrote this they made it so `desc` actually works for action buttons, so too should move everything past the - on this line to the desc var